src/Employer/Employer.Web/Orchestrators/Part1/EmployerOrchestrator.cs

```diff
@@ -47,9 +47,10 @@ public EmployerOrchestrator(IVacancyClient client)
             return vm;
         }
 
-        public async Task<EmployerViewModel> GetLocationViewModelAsync(EmployerEditModel m)
+        public async Task<EmployerViewModel> GetEmployerViewModelAsync(EmployerEditModel m)
         {
-            var vm = await GetLocationViewModelAsync(m.VacancyId);
+            var vrm = new VacancyRouteModel { EmployerAccountId = m.EmployerAccountId, VacancyId = m.VacancyId };
```

Comment:
Wondering if `EmployerEditModel` should inherit from `VacancyRouteModel` then we won't need to new up another object we can just cast the passed in `EmployerEditModel`. 

Or maybe share the same interface?

Reply:
Can we maybe do this as part of a different story, prefer inheritance. I would rather stay away from interface because these will be properties and I would rather an interface declare behavior.